arm9/src/scrcmd_sound.c

```diff
@@ -0,0 +1,299 @@
+#include "global.h"
+#include "script.h"
+#include "scrcmd.h"
+
+extern BOOL FUN_020413C0(struct ScriptContext *ctx);
+extern BOOL FUN_020413E8(struct ScriptContext *ctx);
+extern BOOL FUN_020413F8(struct ScriptContext *ctx);
+extern BOOL FUN_0204140C(struct ScriptContext *ctx);
+extern BOOL FUN_02041424(struct ScriptContext *ctx);
+extern BOOL FUN_0204143C(struct ScriptContext *ctx);
+extern BOOL FUN_02041464(struct ScriptContext *ctx);
+extern BOOL FUN_02041478(struct ScriptContext* ctx);
+extern BOOL FUN_0204149C(struct ScriptContext* ctx);
+extern BOOL FUN_020414B8(struct ScriptContext *ctx);
+extern BOOL FUN_020414CC(struct ScriptContext* ctx);
+extern BOOL FUN_020414E0(struct ScriptContext* ctx);
+extern BOOL FUN_020414FC(struct ScriptContext* ctx);
+extern BOOL FUN_02041518(struct ScriptContext* ctx);
+extern BOOL FUN_02041540(struct ScriptContext* ctx);
+extern BOOL FUN_02041558(struct ScriptContext* ctx);
+extern BOOL ScrCmd_waitcry(struct ScriptContext* ctx);
+extern BOOL FUN_02041598(struct ScriptContext* ctx);
+extern BOOL FUN_020415BC(struct ScriptContext* ctx);
+extern BOOL FUN_020415CC(struct ScriptContext* ctx);
+extern BOOL ScrCmd_playbgm(struct ScriptContext* ctx);
+extern BOOL FUN_020415E0(struct ScriptContext* ctx);
+extern BOOL FUN_02041618(struct ScriptContext* ctx);
+extern BOOL FUN_02041644(struct ScriptContext* ctx);
+extern BOOL FUN_02041650(struct ScriptContext* ctx);
+extern BOOL FUN_02041664(struct ScriptContext* ctx);
+extern BOOL FUN_02041674(struct ScriptContext* ctx);
```

Comment:
These should go into scrcmd.h, starting at line 153 (also make sure to add a comment saying these are from scrcmd_sound)